# Modified from https://gitlab.com/freepascal.org/fpc/source/-/commit/a20a7e3497bccf3415bf47ccc55f133eb9d6d6a0.patch
# FIXME: Remove it when bumping to the next release

--- a/fpcsrc/rtl/linux/aarch64/cprt0.as
+++ b/fpcsrc/rtl/linux/aarch64/cprt0.as
@@ -7,6 +7,18 @@
 	.text
 	.align 2

+	.globl __libc_csu_init
+	.type __libc_csu_init,#function
+	.weak __libc_csu_init
+__libc_csu_init:
+	ret
+
+	.globl __libc_csu_fini
+	.type __libc_csu_fini,#function
+	.weak __libc_csu_fini
+__libc_csu_fini:
+	ret
+
 	.globl _start
 	.type  _start,#function
 _start:
